README.md

```diff
@@ -33,6 +33,102 @@ Download a copy of the TicTacToe repository
 - Once you are in the main root file, enter in the terminal the command: ruby bin/main.
 - Follow the game instructions
 
+## Instructions
+
+ Once the game has started
+ 1. First player place the name
+ 2. First player need to select a symbol from X or O
+ 3. Second player needs to enter the name
+ 4. Symbol for second player is assgined automatically
+ 5. Game starts - Makes player1 to chooose a section to place his symbol (1-9)
```

Comment:
The spelling of 'chooose'. Kindly change that.

Always use tools like Grammarly to check your writings